syntax/pagesetting.php

```diff
@@ -83,4 +87,4 @@ public function render($mode, Doku_Renderer $renderer, $data) {
         return false;
```

Comment:
You can add here a metadata render mode (please see also the development docs at dokuwiki.org)
Next you can use in the action.php the metadata instead of a regexp